screen2.0/src/app/search/ccredetails/ccredetails.tsx

```diff
@@ -18,63 +41,163 @@ type CcreDetailsProps = {
 
 export const CcreDetails: React.FC<CcreDetailsProps> = ({ accession, region, globals, assembly }) => {
   const [value, setValue] = React.useState(0)
+  const [open, setState] = React.useState<boolean>(true)
+
   const handleChange = (_, newValue: number) => {
     setValue(newValue)
   }
+
+  const toggleDrawer = (open) => (event) => {
+    if (event.type === "keydown" && (event.key === "Tab" || event.key === "Shift")) {
+      return
+    }
+    setState(open)
+  }
+
+  // temp theme for toolbar color
+  const theme = createTheme({
```

Comment:
themes can be moved to themes.ts ?

Reply:
got it, moved them to themes.ts in common/lib